lib/lib_i2c/Adafruit_TSL2591_Library/Adafruit_TSL2591.cpp

```diff
@@ -117,7 +117,7 @@ enum
 Adafruit_TSL2591::Adafruit_TSL2591()
 {
   _initialized = false;
-  _integration = TSL2591_INTEGRATIONTIME_100MS;
+  _integration = TSL2591_INTEGRATIONTIME_300MS;
```

Comment:
Modifying an external library in that way is a bad idea as it will overwritten if someone upgrade the library